addon/src/system/scroll-parent.js

```diff
@@ -1,33 +0,0 @@
-function getParentElements(element) {
-  const parentsArray = [];
-
-  if (!element) {
-    return parentsArray;
-  }
-
-  let currentParent = element.parentElement;
-
-  while (currentParent !== null) {
-    parentsArray.push(currentParent);
-    currentParent = currentParent.parentElement;
-  }
-  return parentsArray;
-}
-
-export default function (element) {
-  let position = getComputedStyle(element).position;
-  let excludeStaticParent = position === 'absolute';
-  let scrollParent = getParentElements(element).filter(function (parent) {
-    let parentElemStyles = getComputedStyle(parent);
-    if (excludeStaticParent && parentElemStyles.position === 'static') {
-      return false;
-    }
-    let { overflow, overflowX, overflowY } = parentElemStyles;
-    return /(auto|scroll)/.test(overflow + overflowX + overflowY);
-  })[0];
-
-  if (!scrollParent || scrollParent === document.body) {
-    scrollParent = document;
-  }
-  return position === 'fixed' || scrollParent;
```

Comment:
while TS converting removed `position === 'fixed'` because it makes no sense... the return must be a HTMLElement or Document otherwise code is not correctly in scroll-container.ts because he needs always a dom element https://github.com/adopted-ember-addons/ember-sortable/pull/581/files#diff-49b1162a2852f7a45dc24d5027bc3ea12f95d37caef8c2b020f7709803bb01a6R24